tpot/driver.py

```diff
@@ -351,6 +351,18 @@ def _get_arg_parser():
     )
 
 
+    parser.add_argument(
+        '-memory',
+        action='store',
+        dest='MEMORY',
+        default=None,
+        type=str,
+        help=(
+            'Path of a directory for memory caching or \"auto\" for using temporary '
```

Comment:
Clarify that this is pipeline caching.

Reply:
OK, docs were added in the commit below.